backend-worker/worker.py

```diff
@@ -1,26 +1,546 @@
+            -10965275.57, 3429693.30)
+
+        # Raster units are in meters (mercator)
+        parcel = point_over_san_antonio.buffer(100)
+        pygeoprocessing.geoprocessing.shapely_geometry_to_vector(
+            [point_over_san_antonio, parcel],
+            os.path.join(self.workspace_dir, 'parcel.shp'),
+            _WEB_MERCATOR_SRS.ExportToWkt(), 'ESRI Shapefile')
+
+        _create_new_lulc(parcel.wkt, gtiff_path)
+
+        raster_info = pygeoprocessing.get_raster_info(gtiff_path)
+
+        raster_bbox = shapely.geometry.box(*raster_info['bounding_box'])
+        epsg3857_raster_bbox = ogr.CreateGeometryFromWkt(raster_bbox.wkt)
+        epsg3857_raster_bbox.Transform(ALBERS_EQ_AREA_TO_WEB_MERCATOR)
+        epsg3857_raster_bbox_shapely = shapely.wkt.loads(
+            epsg3857_raster_bbox.ExportToWkt())
+
+        self.assertTrue(epsg3857_raster_bbox_shapely.contains(parcel))
+
+    def test_fill(self):
+        # University of Texas: San Antonio, selected by hand in QGIS
+        # Coordinates are in EPSG:3857 "Web Mercator"
+        point_over_san_antonio = shapely.geometry.Point(
+            -10965275.57, 3429693.30)
+        parcel = point_over_san_antonio.buffer(100)
+
+        target_raster_path = os.path.join(self.workspace_dir, 'raster.tif')
+        fill_parcel(parcel.wkt, 15, target_raster_path)
+
+        result_array = pygeoprocessing.raster_to_numpy_array(
+                target_raster_path)
+        self.assertEqual(
+            numpy.sum(result_array[result_array != NLUD_NODATA]), 600)
+        self.assertEqual(numpy.sum(result_array == 15), 40)
+
+    def test_wallpaper(self):
+        # University of Texas: San Antonio, selected by hand in QGIS
+        # Coordinates are in EPSG:3857 "Web Mercator"
+        point_over_san_antonio = shapely.geometry.Point(
+            -10965275.57, 3429693.30)
+        parcel = point_over_san_antonio.buffer(100)
+
+        # Apache Creek, urban residential area, San Antonio, TX.
+        # Selected by hand in QGIS.  Coordinates are in EPSG:3857 "Web
+        # Mercator"
+        pattern = shapely.geometry.box(
+            *shapely.geometry.Point(
+                -10968418.16, 3429347.98).buffer(100).bounds)
+
+        target_raster_path = os.path.join(
+            self.workspace_dir, 'wallpapered_raster.tif')
+
+        nlud_path = 'appdata/nlud.tif'
+        wallpaper_parcel(parcel.wkt, pattern.wkt, nlud_path,
+                         target_raster_path, self.workspace_dir)
+
+        # This is useful for debugging
+        # import pdb; pdb.set_trace()  # print(self.workspace_dir)
+
+
+def _reproject_to_nlud(parcel_wkt_epsg3857):
+    ogr_geom = ogr.CreateGeometryFromWkt(parcel_wkt_epsg3857)
+    err_code = ogr_geom.Transform(WEB_MERCATOR_TO_ALBERS_EQ_AREA)
+    if err_code:
+        LOGGER.warning(
+            "Transformation failed on parcel; continuing with "
+            f"{parcel_wkt_epsg3857}")
+    assert ogr_geom.ExportToWkt() != parcel_wkt_epsg3857
```

Comment:
What happens if this `assert` fails? Are we just checking that some transformation has occurred?